seed/views/v3/data_quality_check_rules.py

```diff
@@ -0,0 +1,79 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+"""
+import logging
+
+from django.utils.decorators import method_decorator
+
+from drf_yasg.utils import swagger_auto_schema, no_body
+
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework.mixins import ListModelMixin
+
+from seed.decorators import ajax_request_class
+from seed.lib.superperms.orgs.decorators import has_perm_class
+from seed.models.data_quality import DataQualityCheck, Rule
+from seed.serializers.rules import RuleSerializer
+from seed.utils.api import api_endpoint_class
+from seed.utils.api_schema import AutoSchemaHelper
+from seed.utils.viewsets import UpdateWithoutPatchModelMixin
+
+_log = logging.getLogger(__name__)
+
+
+nested_org_id_path_field = AutoSchemaHelper.base_field(
+    "nested_organization_id",
+    "IN_PATH",
+    "Organization ID - identifier used to specify a DataQualityCheck and its Rules",
+    True,
+    "TYPE_INTEGER"
+)
+
+
+@method_decorator(
+    name='list',
+    decorator=swagger_auto_schema(manual_parameters=[nested_org_id_path_field])
+)
+@method_decorator(
+    name='update',
+    decorator=swagger_auto_schema(manual_parameters=[nested_org_id_path_field])
+)
+class DataQualityCheckRuleViewSet(viewsets.GenericViewSet, ListModelMixin, UpdateWithoutPatchModelMixin):
+    serializer_class = RuleSerializer
+    model = Rule
+    pagination_class = None
+
+    def get_queryset(self):
+        # Handle the anonymous case (e.g. Swagger page load)
```

Comment:
Swagger page was loading with an error/warning, and this was the fix: https://github.com/carltongibson/django-filter/issues/966